### PR TITLE
SELV3-553: Create table on dhis2 page

### PR DESCRIPTION
**Pull request by mdulko**

### CODE REVIEW

File: src/admin-dhis2/admin-dhis2.routes.js
Comment (NikolaLaskowska) [inline src/admin-dhis2/admin-dhis2.routes.js:29]: this should be false, we don't support offline mode for DHIS2

Comment (mdulko) [reply src/admin-dhis2/admin-dhis2.routes.js:29]: Sure, thanks for the information about that, I have set up as true and thought this one needs to be true, will update!

File: src/admin-dhis2/components/AdminDhis2Page/AdminDhis2Page.jsx
Comment (NikolaLaskowska) [inline src/admin-dhis2/components/AdminDhis2Page/AdminDhis2Page.jsx:18]: Maybe we should rename this component to `AdminDhis2ServersPage` since there will be only server data? @sniedzielski what do you think?

File: src/admin-dhis2/admin-dhis2.service.js
Comment (sniedzielski) [inline src/admin-dhis2/admin-dhis2.service.js:39]: I wonder if we can precise what kind of 'get' method is this one by renaming it. For example 'getServerConfig' etc. @mdulko

File: src/admin-dhis2/components/AdminDhis2Page/AdminDhis2Page.jsx
Comment (sniedzielski) [inline src/admin-dhis2/components/AdminDhis2Page/AdminDhis2Page.jsx:41]: Can you rename this one:
Name -> name?
Please do this to have consistent naming conventions.

File: src/admin-dhis2/components/AdminDhis2Page/AdminDhis2Page.jsx
Comment (sniedzielski) [inline src/admin-dhis2/components/AdminDhis2Page/AdminDhis2Page.jsx:42]: Same as above. Can you rename this one:
URL -> url?

File: src/admin-dhis2/components/AdminDhis2Page/AdminDhis2Page.jsx
Comment (sniedzielski) [inline src/admin-dhis2/components/AdminDhis2Page/AdminDhis2Page.jsx:55]: If you apply my suggestion from line 41 change please change 'accessor' value into 'name'.

**Review by sniedzielski — COMMENTED** [review]
@mdulko I left few minor comments. Please refer to them.

File: src/admin-dhis2/components/AdminDhis2Page/AdminDhis2Page.jsx
Comment (sniedzielski) [reply src/admin-dhis2/components/AdminDhis2Page/AdminDhis2Page.jsx:18]: That is a good idea. This new name will be better and more precise the purpose of this page.

File: src/admin-dhis2/admin-dhis2.service.js
Comment (mdulko) [reply src/admin-dhis2/admin-dhis2.service.js:39]: sure, I think that fits nicely there

File: src/admin-dhis2/components/AdminDhis2Page/AdminDhis2Page.jsx
Comment (mdulko) [reply src/admin-dhis2/components/AdminDhis2Page/AdminDhis2Page.jsx:41]: sure will rename the whole conventions

File: src/admin-dhis2/components/AdminDhis2Page/AdminDhis2Page.jsx
Comment (mdulko) [reply src/admin-dhis2/components/AdminDhis2Page/AdminDhis2Page.jsx]: updated

File: src/admin-dhis2/components/AdminDhis2Page/AdminDhis2Page.jsx
Comment (mdulko) [reply src/admin-dhis2/components/AdminDhis2Page/AdminDhis2Page.jsx]: updated

File: src/admin-dhis2/components/AdminDhis2Page/AdminDhis2Page.jsx
Comment (mdulko) [reply src/admin-dhis2/components/AdminDhis2Page/AdminDhis2Page.jsx]: updated

**Review by sniedzielski — APPROVED** [review]
LGTM after those recent changes after my review @mdulko. Let's wait for Nikola's feedback.

**Pull request merged**
